lib/flapjack/gateways/jsonapi/metrics_methods.rb

```diff
@@ -30,10 +30,16 @@ def event_queue_length
             redis.llen('events')
           end
 
+          def event_queue
+            { 'length' => redis.llen('events') }
+          end
+
           def processed_events
```

Comment:
Hmm. I've documented these two endpoints separately as follows:

- [event_queue](http://flapjack.io/docs/1.0/jsonapi/?ruby#event-queue)
- [processed_events](http://flapjack.io/docs/1.0/jsonapi/?ruby#processed-events)

The event_queue endpoint could have other things added in the future, but maybe you're right. Would you combine them like this perhaps? 

## GET /metrics/events
```json
{
  "events": 
  {
    "processed": {
      "all": 153,
      "failure": 104,
      "ok": 12,
      "invalid": 29,
      "action": 8
    }
    "queue": {
      "length": 0
    }
  }
}
```

I'm not sure it really give us anything.